src/OrbitGl/AccessibleTimeGraph.cpp

```diff
@@ -30,7 +30,9 @@ int TimeGraphAccessibility::AccessibleChildCount() const {
 }
 
 const AccessibleInterface* TimeGraphAccessibility::AccessibleChild(int index) const {
-  return time_graph_->GetTrackManager()->GetVisibleTracks()[index]->AccessibilityInterface();
+  return time_graph_->GetTrackManager()
```

Comment:
Why is this reformatted? Was that intended?

Reply:
Yes, the function name changed from `AccessibilityInterface` to `GetOrCreateAccessibilityInterface`